src/ai/backend/agent/server.py

```diff
@@ -556,6 +557,39 @@ async def commit(
             "path": str(Path(subdir, filename)),
         }
 
+    @rpc_function
+    @collect_error
+    async def get_local_config(self):
+        def get_or_empty(val):
+            if val is None:
+                return ""
+            return str(val)
+
+        return {
+            "agent": {
+                "abuse-report-path": get_or_empty(
+                    self.local_config["agent"].get("abuse-report-path")
+                ),
+                "image-commit-path": get_or_empty(
+                    self.local_config["agent"].get("image-commit-path")
+                ),
+            },
+            "watcher": self.local_config["watcher"],
+        }
+
+    @rpc_function
+    @collect_error
+    async def get_abusing_report(
+        self,
+        kernel_id,  # type: str
+    ):
+        if (abuse_path := self.local_config["agent"].get("abuse-report-path")) is not None:
+            report_path = Path(abuse_path, f"report.{kernel_id}.json")
+            if report_path.is_file():
+                with open(report_path, "r") as file:
+                    return json.load(file)
+        return
```

Comment:
`return` here seems meaningless. If you were to return `None`, please write it down explicitly.
```suggestion
        return None
```